skateboard.py

```diff
@@ -2,23 +2,24 @@
 from scipy.integrate import odeint as edo
```

Comment:
Le commentaire de ce commit n'est pas vraiment idéal...

Reply:
```suggestion
from scipy.integrate import odeint as edo
```
ce serait plus adéquat de laisser odeint ?